### PR TITLE
Calculate audio energy and send in RTP header extension

### LINKED ISSUE #614: Add support for ssrc-audio-level header extension
Currently there is some support for this in aiortc, but it's not fully wired up. I believe the required work is:
1) add "urn:ietf:params:rtp-hdrext:ssrc-audio-level" to HEADER_EXTENSIONS in __init__.py
2) return audio frame energy from `_next_encoded_frame` and pass it to `packet.extensions.audio_level`

I can take a stab at this if there's no planned work here.

### PR DESCRIPTION
**Pull request by juberti**
Fixes #614

### CODE REVIEW

**Comment by juberti** [comment]
@jlaine

**Comment by jlaine** [comment]
Hi @juberti thanks for the PR!

Some notes:

- I've let CI run and it looks like the linters are not happy
- maybe it would make sense to pull the `_compute_level_dbov` method out to a plain function (for instance in `aiortc.rtp`) so that we can have a unit test with some test vectors?
- I see `level` is set to `None` for video frames, isn't this going to blow up: `packet.extensions.audio_level = (False, -enc_frame.level)`
- how costly is this calculation?

File: src/aiortc/rtcrtpsender.py [hunk 1]
```diff
@@ -36,12 +38,19 @@
...
 
 RTP_HISTORY_SIZE = 128
 RTT_ALPHA = 0.85
 
-
+class RTCEncodedFrame:
+    def __init__(self, payloads: List[bytes], timestamp: int, level: int):
+        self.payloads = payloads
+        self.timestamp = timestamp
+        self.level = level
```
Comment (jlaine) [inline src/aiortc/rtcrtpsender.py:52]: Maybe `audio_level` would be more explicit?

**Comment by juberti** [comment]
OK, fixed the lint issues and the video-frame issue, and broke out the energy calc function and added test for it. 

As for costs, it's small but not nothing, maybe 3% of a core on my system. numpy might be an option if this is too much, although there might be other costs incurred roundtripping to numpy and back.

**Comment by jlaine** [comment]
I ran `isort` on the code which fixes additional linter errors.

**Comment by juberti** [comment]
Thanks for doing that, I had run flake8 and black but not isort.

**Comment by jlaine** [comment]
For my information, does this header need to be negotiated or are we good to always send it?

**Comment by juberti** [comment]
It should be negotiated, that's right. I think there is already logic to only send the header if it's been negotiated, although it will always be computed.

**Comment by jlaine** [comment]
OK, I'm going to merge this PR as is, even though I'm a bit concerned by the computational overhead (some of aiortc's usecase are on low power devices).

If issues suddenly get reported related to performance, would you consider working on a follow up PR to make this extension optional? I noticed that webrtc-extensions mentions a `RTCRtpTransceiver.setOfferedRtpHeaderExtensions` method, maybe that could be used?

**Comment by jlaine** [comment]
Oops, I forgot to squash the commits, so I've had to force-push to `main`..

**Comment by juberti** [comment]
Will you be publishing a new rev of aiortc with these changes?

**Pull request merged**